### PR TITLE
Update and fix sizing for hubble data dicts

### PR DESCRIPTION
**Pull request by chowbao**
Updates and table formatting and organization changes for Hubble data dicts

### CODE REVIEW

**Comment by chowbao** [comment]
* Data dicts content has been copied over
* Table css simplified and formats the table a little better

**Review by sydneynotthecity — COMMENTED** [review]
nit: can we drop the Primary Keys from all of the Table Metadata? Most of these tables do not have primary keys, and I'd like consistency across all tables. 

Also the `history_assets` table is missing Natural Keys if you don't mind filling those in

**Comment by chowbao** [comment]
> nit: can we drop the Primary Keys from all of the Table Metadata? Most of these tables do not have primary keys, and I'd like consistency across all tables.
> 
> Also the `history_assets` table is missing Natural Keys if you don't mind filling those in

Updated in f1268f44e77c7f8c52ace97f04d5fed909d5e0d9 and 4f192054342095a294579af88d539cf182ae4ee5

There were actually a lot of issues with the table metadata lol
* Reduced the natural keys to actual unique identifier of data by using `closed_at`
* Fixed the dbt docs link to use the public dbt docs instead of the private one
* Fixed incorrect partition fields
* Fixed cluster ordering

**Review by sydneynotthecity — APPROVED** [review]
Nothing blocking the merge, looks good

File: docs/data/hubble/data-catalog/data-dictionary/history-trades.mdx [hunk 1]
```diff
@@ -1,34 +1,50 @@
...
-| seller_is_exact | Indicates whether the buying or selling party trade was impacted by rounding slippage. If true, the buyer was impacted. If false, the seller was impacted | boolean |  |  |  |  | No |  |
-| selling_asset_id | Unique identifier for selling_asset_code, selling_asset_issuer | integer |  |  |  | cluster | No |  |
-| buying_asset_id | Unique identifier for buying_asset_code, buying_asset_issuer | integer |  |  |  | cluster | No |  |
+<div className="scoped-data-dictionary-table">
+
+## Table Metadata
+
+| Property | Configuration |
+| --- | --- |
+| Natural Key(s) | history_operation_id, order, ledger_closed_at |
```
[the remark this answers is not in the document]
Comment (chowbao) [reply docs/data/hubble/data-catalog/data-dictionary/history-trades.mdx:13]: Ah that's right

**Pull request merged**